src/clojure/nrepl/core.clj

```diff
@@ -219,6 +221,8 @@
                            (merge connect-defaults
                                   (socket-info uri))))))
 
+(add-socket-connect-method! "edn" {:transport-fn transport/nrepl+edn
```

Comment:
I actually wanted us to name the protocol `nrepl+edn` and the function itself `edn`.

Reply:
Ok. So swap those two around? Will do.

Reply:
👍